app/frontend/styles/components/location_finder.scss

```diff
@@ -9,11 +9,11 @@
     width: 100%;
   }
 
-  .govuk-c-loader__spinner {
-    left: -3px;
-    position: relative;
-    top: -3px;
-  }
+  // .govuk-c-loader__spinner {
```

Comment:
think i left in this commented out code but it should be removed altogether

Reply:
Will remove it now